crypto/signer.go

```diff
@@ -1,11 +1,18 @@
 package neofscrypto
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/nspcc-dev/neofs-api-go/v2/refs"
 )
 
+// ErrIncorrectSigner is returned from function when the signer passed to it
+// is incompatible with the function requirements. This variable is intended
+// to be used as documentation and for [errors.Is] purposes and MUST NOT be
+// changed.
+var ErrIncorrectSigner = errors.New("incorrect signer")
```

Comment:
if it's exported I expect to see this error in docs, e.g. in introduced `client.New`, otherwise it doesn't make sense to expose it imo

Reply:
Fixed.